Documentation/Signoff/Vehicle Lights.md

```diff
@@ -0,0 +1,138 @@
+# Vehicle Lights signoff
+## Function of subsystem 
+The function of this subsystem is to allow users to interact with the car Headlights while also learning the concepts of electrical components and code. This subsystem will take different inputs at the same time from the input breadboard and Coding subsystems. It will use a microcontroller to take 4 inputs from those subsystems and send them to a multiplexer. The multiplexer will have an enable and two select lines that control what input is sent on through to the LEDs. The user will use inputs such as switches within the input breadboard subsystem to control the enable and select lines. 
+
+## Constraints
+### Derived from Shall statements
+
+C1: The Kit shall operate at safe and compatible voltage levels for educational purposes.
+- The kit needs to operate at safe voltages but also make sure that the voltages are compatible with the Arduino and other components used in this subsystem since ratings and compatability can be different. 
+
+
+     
+
+C2: The kit shall ensure proper safety features for any connections that are made making sure they are connected properly and there are limitations for current so there is no overcurrent for safety reasons.
+- All Arduino and soldered components need to be safely and properly connected. The soldered components must be soldered correctly and firmly. Any wires used to connect the Arduino to the multiplexer and LEDs that are soldered also need to be properly connected for safety protection. Any LEDs or other current-limited components will need to have current-limiting resistors to prevent over-current. 
+
+
+C3 & C9. The kit shall contain clear instructions, labeled components, user-friendly interface, and comprehensive materials such as a kit manual.
```

Comment:
How do you decide if something is or is not clearly labeled? (I'm being a bit pedantic here. But what I'm trying to get at is, a constraint needs to be precise enough that you can clearly show, through analysis, that it is or is not met.)